input/resources/au-core-medication.xml

```diff
@@ -69,6 +69,11 @@
       <path value="Medication.code.coding"/>
       <sliceName value="pbs"/>
       <mustSupport value="true"/>
+      <max value="1" />
```

Comment:
Max  value of 1 is not appropriate for the terminology slices in AU Core Medication and AU Core MedicationRequest - AU Base does not restrict to be max of 1.

Reply:
Fixed